src/sysinfo

```diff
@@ -1207,6 +1258,7 @@ get_smartos_vnics
 get_aggr_params
 get_boot_time
 get_dc_info
+get_or_store_uuid
```

Comment:
Also, should this go in the GZ-only section above (say near line 1237 after get_or_store_uuid)?